.github/.markdownlintignore

```diff
@@ -1,5 +1,5 @@
 ansible_collections/arista/avd/molecule/
 ansible_collections/arista/avd/examples/*/documentation/
 ansible_collections/arista/avd/tests/unit/filters/toc_files/
-ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/Input\ Variables.md
+ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/
```

Comment:
do we still need this, after the links have been removed?

Reply:
We do, I need to figure out one last error. I'm not sure why aliases doesn't include one blank line around the code block. It's the last error to to pass and we can remove that line and run checks on all the auto created schema docs

```shell
Check for Linting errors on MarkDown files...............................Failed
- hook id: markdownlint
- exit code: 1

ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/data_model/Aliases.md:18 MD031/blanks-around-fences Fenced code blocks should be surrounded by blank lines [Context: "```"]
```